src/components/KDateTimePicker/KDateTimePicker.spec.ts

```diff
@@ -0,0 +1,237 @@
+import { mount } from '@cypress/vue'
+import {
+  format,
+  // roundToNearestMinutes
+} from 'date-fns'
+import { TimePeriods, TimeframeKeys } from '@/utils/KDatePickerMockData'
+import KDateTimePicker from '@/components/KDateTimePicker/KDateTimePicker.vue'
+
+const exampleTimeFrames = [
+  {
+    section: 'Last',
+    values: [
+      TimePeriods.get(TimeframeKeys.FIFTEEN_MIN),
+      TimePeriods.get(TimeframeKeys.ONE_HOUR),
+      TimePeriods.get(TimeframeKeys.THREE_HOUR),
+      TimePeriods.get(TimeframeKeys.SIX_HOUR),
+      TimePeriods.get(TimeframeKeys.TWELVE_HOUR),
+      TimePeriods.get(TimeframeKeys.ONE_DAY),
+      TimePeriods.get(TimeframeKeys.SEVEN_DAY),
+      TimePeriods.get(TimeframeKeys.THIRTY_DAY),
+    ],
+  },
+  {
+    section: 'Current',
+    values: [
+      TimePeriods.get(TimeframeKeys.CURRENT_WEEK),
+      TimePeriods.get(TimeframeKeys.CURRENT_MONTH),
+    ],
+  },
+  {
+    section: 'Previous',
+    values: [
+      TimePeriods.get(TimeframeKeys.PREVIOUS_WEEK),
+      TimePeriods.get(TimeframeKeys.PREVIOUS_MONTH),
+    ],
+  },
+]
+
+const today = new Date() // eg: 'Thu Sep 08 2022 13:03:28 GMT-0700 (Pacific Daylight Time)'
+const todayDateString = format(new Date(today), 'PP')
+const todayDateTimeString = format(new Date(today), 'PP hh:mm a')
+const twoDaysAgo = new Date(today.getTime() - (2 * 24 * 60 * 60 * 1000))
+const minDate = new Date(today.getTime() - (365 * 24 * 60 * 60 * 1000))
+const maxDate = today
+const defaulTimeRange = {
```

Comment:
This is spelled incorrectly; can you update and fix the usage?
```suggestion
const defaultTimeRange = {
```

Reply:
:doh: